### PR TITLE
Add Kafka message timestamp

### PR DESCRIPTION
**Pull request by abbccdda**

### CODE REVIEW

**Comment by sbaldwin-rs** [comment]
Tests are failing

File: src/main/java/rockset/models/KafkaMessage.java [hunk 1]
```diff
@@ -15,6 +15,7 @@ public class KafkaMessage {
   @SerializedName("partition") public int partition;
   @SerializedName("offset") public long offset;
   @SerializedName("key") public Object key;
+  @SerializedName("timestamp") public long timestamp;
```
Comment (kwadhwa18) [inline src/main/java/rockset/models/KafkaMessage.java:18]: we need changes in https://github.com/rockset/rs/blob/1a4e2dd8b96a3df23a41a4377b98fb34e9f63071/java/rockset-main/src/main/java/io/rockset/main/api/models/documents/KafkaMessage.java#L7-L8 too.

And I don't think we can change the kafka-connect code until receiver understands this new field

**Review by kwadhwa18 — CHANGES_REQUESTED** [review]

File: src/main/java/rockset/models/KafkaMessage.java (hunk 1 above)
Comment (abbccdda) [reply src/main/java/rockset/models/KafkaMessage.java:18]: Shouldn't we have connect populate this field first as it is upstream? You mean there will be a parse error?

Comment (kwadhwa18) [reply src/main/java/rockset/models/KafkaMessage.java:18]: receiver will reject unknown fields from what I know. Should be easy to verify

**Pull request closed without merging**